diff --git a/node_modules/@studio123/strapi-plugin-image-color-palette/server/services/image-color-palette.js b/node_modules/@studio123/strapi-plugin-image-color-palette/server/services/image-color-palette.js
index 66d105c..5f31e13 100644
--- a/node_modules/@studio123/strapi-plugin-image-color-palette/server/services/image-color-palette.js
+++ b/node_modules/@studio123/strapi-plugin-image-color-palette/server/services/image-color-palette.js
@@ -28,6 +28,7 @@ const convert = (rgbObj, format) => {
 module.exports = ({ strapi }) => ({
     async generate(url, mime) {
         try {
+            url = `https://strapi.mewapi.io${url}`
             const settings = getService(strapi, 'settings').get();
 
             let dominantColor = null;
